homeassistant/helpers/template.py

```diff
@@ -335,10 +335,10 @@ def _resolve_state(self, entity_id_or_state):
         return None
 
 
-def forgiving_round(value, precision=0):
+def forgiving_round(value, precision=0, method='common'):
     """Rounding filter that accepts strings."""
     try:
-        value = round(float(value), precision)
+        value = round(float(value), precision, method)
```

Comment:
This is not the jinja2 round. This is the Python round and `method` is not part of the API: https://docs.python.org/3.5/library/functions.html#round